srcpkgs/walk/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'walk'
+pkgname=walk
+version=20190920
+revision=1
+_commit=70f7a8c104acf109ae810a9b34eb8e8b92b4d27d
```

Comment:
Consider opening an issue upstream to periodically tag actual releases?

Reply:
Upstream seemst to not have moved since 2019... Unless this is widely requested or similar, I don't fancy accepting something that doesn't have any official releases.

Reply:
I asked for that in https://github.com/google/walk/issues/3